cmd/acikubectl/cmd/debug.go

```diff
@@ -190,36 +227,37 @@ func clusterReport(cmd *cobra.Command, args []string) {
 	gzWriter := gzip.NewWriter(outfile)
 	tarWriter := tar.NewWriter(gzWriter)
 
-	now := time.Now()
-	hasErrors := false
-	for _, cmd := range cmds {
-		buffer := new(bytes.Buffer)
+	var wg sync.WaitGroup
+	wg.Add(len(cmds))
 
-		fmt.Fprintln(os.Stderr, "Running command: kubectl", cmd.args)
-		err = execKubectl(cmd.args, buffer)
-		if err != nil {
-			fmt.Fprintln(os.Stderr, err)
-			hasErrors = true
-			continue
-		}
+	// Execute kubectl commands
+	for _, cmd := range cmds {
+		go func(cmd reportCmdElem, tarWriter *tar.Writer){
+			buffer := new(bytes.Buffer)
+			now := time.Now()
+			defer wg.Done()
+
+			fmt.Fprintln(os.Stderr, "Running command: kubectl", strings.Join(cmd.args, " "))
```

Comment:
Request: <srings> pkg seems to be getting used only in this "one line (Join)" in the entire file.  Can we avoid that?
Possibly if we construct the args with single element then this join can be avoided, somewhat like this:
args: []string{"describe nodes -n " + systemNamespace}
args: []string{"describe daemonset aci-containers-host -n " + systemNamespace}

Reply:
I meant the "strings" pkg